### DISCUSSION TITLE
Alien door of Alien_GeneticStorage1R state desync in dedicated server

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15422] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

it's still close in client side when button is pressed
but I can pass through so it's actually open

### Reproduction steps

1. test the module in dedicated server?

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
![20241221162153](https://github.com/user-attachments/assets/2fde5fb0-1d54-4fc5-a8cc-27501d292dcb)

**Comment by Jasontti** [comment]
Tested a fix for this, buttons have been changed to toggles.
Closing discussion as resolved.
